client/components/photo/photo.tsx

```diff
@@ -0,0 +1,11 @@
+import Image from 'next/image';
+
+export const Photo = ({ img }: { img: string }) => {
+  if (!img) return null;
+
+  return (
+    <div className="fixed top-4 right-4 border border-gray-300 shadow-md">
+      <Image id="image" alt="Received image" src={img} width={400} height={400} />
```

Comment:
noticed these warnings in the console with the Image tag from next js...
![image](https://github.com/user-attachments/assets/d7d5c7f5-389a-4a31-9e83-f6006fa7739c)

i think it's because our version of next is behind since the latest version doesn't work with rpi...?
would you be able to change `Image` to `img` tag and do the same in the `backend.tsx` file?

Reply:
please refer to the newest commit

Reply:
looks good! not seeing the warning on my side -- ty for the fix!